packages/dx-react-grid-bootstrap3/src/templates/table-no-data-cell.jsx

```diff
@@ -38,7 +24,7 @@ export class TableNoDataCell extends React.PureComponent {
         <div
           style={{
             display: 'inline-block',
-            position: stickyPosition,
+            position: 'fixed',
```

Comment:
We can extract it into constant

Reply:
What does you mean? It is normal CSS value.

Reply:
```js
const style = {
  display: 'inline-block',
  position: 'fixed',
  // and so on...
};

// ...

<div
  style={style}
>
```

Reply:
Should we do that? We use that style once, in this `div`

Reply:
We create a new object on each render. We often try to avoid similar cases.